lib/index.js

```diff
@@ -164,7 +167,7 @@ Appboy.prototype.initializeV2 = function(customEndpoint) {
   var self = this;
   if (userId) {
     this.load('v2', function() {
-      window.appboy.changeUser(userId, function() {
+      self._changeUser(userId, function() {
```

Comment:
Seems fine to do this, though as long as the client is using `analytics.ready`, they should be safe with `window.appboy.changeUser` anyways as we won't be ready until then. I think this is probably the more defensive approach though 👍